libraries/radwordsprocessing/formats-and-conversion/word-file-formats/doc/docformatprovider.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Using DocFormatProvider
+page_title: Using DocFormatProvider
+description: Import from Word 97 - Word 2003 Binary file format Using DocFormatProvider
+slug: radwordsprocessing-formats-and-conversion-doc-docformatprovider
+tags: using,docformatprovider
+published: True
+position: 1
+---
+
+# Doc
+
+DOC is the default target Binary file format developed by Microsoft for representing documents in Word 97 - Word 2003 and is one of the supported formats by RadWordsProcessing.
+
+# Dot
+
+DOT is а Word 97 - Word 2003 Binary file format, developed by Microsoft for representing document templates and is one of the supported formats by RadWordsProcessing.
+
+# Using DocFormatProvider
+
+**DocFormatProvider** makes it easy to import a **RadFlowDocument** from DOC and DOT format, preserving the entire document structure and formatting.
```

Comment:
**DocFormatProvider** makes it easy to import DOC or DOT files into a **RadFlowDocument**, preserving the entire document structure and formatting.